receiver/postgresqlreceiver/testdata/config.yaml

```
@@ -0,0 +1,27 @@
+receivers:
+  postgresql:
+    endpoint: localhost:5432
+    transport: tcp
+    username: otel
+    password: $POSTGRESQL_PASSWORD
+    databases:
+      - otel
+    collection_interval: 10s
+    insecure: false
+    unsecure_skip_verify: false
+    ca_file: /home/otel/authorities.crt
+    cert_file: /home/otel/mypostgrescert.crt
+    key_file: /home/otel/mypostgreskey.key
```

Since this is such a verbose config, I suggest we show two configurations here - one just like this, and one with only the required parameters, or at least just the most common ones.

```suggestion
  postgresql:
    endpoint: localhost:5432
    username: otel
    password: $POSTGRESQL_PASSWORD
  postgresql/all:
    endpoint: localhost:5432
    transport: tcp
    username: otel
    password: $POSTGRESQL_PASSWORD
    databases:
      - otel
    collection_interval: 10s
    insecure: false
    unsecure_skip_verify: false
    ca_file: /home/otel/authorities.crt
    cert_file: /home/otel/mypostgrescert.crt
    key_file: /home/otel/mypostgreskey.key
```

---

Here's another example of this pattern: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/testdata/config.yaml